drivers/flash/flash_stm32h7x.c

```diff
@@ -861,6 +861,16 @@ static const struct flash_parameters *flash_stm32h7_get_parameters(const struct
 	return &flash_stm32h7_parameters;
 }
 
+/* Gives the total logical device size in bytes and return 0. */
+static int flash_stm32h7_get_size(const struct device *dev, uint64_t *size)
+{
+	ARG_UNUSED(dev);
+
+	*size = (uint64_t)(LL_GetFlashSize() * 1024);
```

Comment:
Sorry, I was not explicit. My comment applies also here and in **drivers/flash/flash_stm32wb0x.c** and **drivers/flash/flash_stm32wba_fm.c**.

Reply:
Ok, modified foe all other series